day-2.lisp

```diff
@@ -0,0 +1,25 @@
+(defpackage aoc-day-2
+  (:use :cl)
+  (:export :answers))
+(in-package :aoc-day-2)
+
+(defun answers (input-file)
+  (let ((input (mapcar
+                   (lambda (s) (sort (mapcar 'parse-integer (uiop:split-string s :separator "x")) '<))
```

Comment:
How does lisp failures for parse-integer here, is it similar to filter_map in rust where null values aren't present int he final collection?

Reply:
I believe it just raises an error. It's kind of like throwing an exception. It can be ignored if wrapped with `ignore-errors` or caught and handled if desired. See: https://lispcookbook.github.io/cl-cookbook/error_handling.html

Currently, this error would go uncaught and result in a fatal error.